test/views/github-dotcom-markdown.test.js

```diff
@@ -140,6 +140,7 @@ describe('GithubDotcomMarkdown', function() {
     });
 
     it('opens item in pane and activates accordingly', async function() {
+      atom.workspace = {open: () => {}};
```

Comment:
Huh. I'm curious why this one was causing problems?

Reply:
@smashwilson I am not sure why `atom.workspace` is not being initialized here. This would need further investigations.